src/tls13utils.rs

```diff
@@ -520,22 +566,28 @@ pub(crate) fn check_lbytes3_full(b: &Bytes) -> Result<(), TLSError> {
 // Handshake Data
 pub struct HandshakeData(pub Bytes);
 
-pub(crate) fn handshake_data(b: Bytes) -> HandshakeData {
-    HandshakeData(b)
-}
-pub(crate) fn handshake_data_bytes(hd: &HandshakeData) -> Bytes {
-    hd.0.clone()
-}
+impl HandshakeData {
+    pub(crate) fn len(&self) -> usize {
+        self.0.len()
+    }
 
-pub(crate) fn handshake_data_len(p: &HandshakeData) -> usize {
-    p.0.len()
+    pub(crate) fn bytes(&self) -> Bytes {
```

Comment:
By Rust naming conventions this should be `to_bytes`.

https://rust-lang.github.io/api-guidelines/naming.html